### ISSUE TITLE
Bug: Tekton Pipeline task `scan-image` failing for mixed case projects

### ISSUE THREAD

**Issue by mtamboli** [kabanero-io/kabanero-pipelines#212]
If the project name is mixed case, image scan task  of the pipleline run fails.
```
Pulling image image-registry.openshift-image-registry.svc:5000/kabanero/appsodyMPversion:9806846206d20001e5aa847a49e579392ee18e7a
error parsing reference to image "image-registry.openshift-image-registry.svc:5000/kabanero/appsodyMPversion:9806846206d20001e5aa847a49e579392ee18e7a": error parsing image name "image-registry.openshift-image-registry.svc:5000/kabanero/appsodyMPversion:9806846206d20001e5aa847a49e579392ee18e7a": invalid reference format: repository name must be lowercase
```

**Comment by aadeshpa** [comment]
The pull request for this issue fix is : https://github.com/kabanero-io/kabanero-pipelines/pull/214

**Comment by kvijai82** [comment]
First part of the fix is in RC1.  A followup PR to not change the case of
the tag will be included in RC2

**Comment by kvijai82** [comment]
The fix is in 0.6.0 rc2. @mtamboli please close this issue once you have validated.  Thanks!

**Comment by mtamboli** [comment]
This is working in rc2. Closing it.
